config/v1/types_ingress.go

```diff
@@ -20,7 +20,8 @@ type Ingress struct {
 }
 
 type IngressSpec struct {
-	// default suffix.  It goes here or it gets removed from server
+	// Domain specifies the default subdomain for routes.
```

Comment:
lower "d" in API docs to match the json string.

Reply:
Also is it common knowledge for OpenShift users that domain for ingress means subdomain?